internal/e2e/status_test.go

```diff
@@ -102,31 +104,34 @@ func Test_ComponentStatusReporting_SharedInstance(t *testing.T) {
 	err = s.Shutdown(context.Background())
 	require.NoError(t, err)
 
-	assert.Equal(t, 5, len(eventsReceived))
+	require.Equal(t, 2, len(eventsReceived))
```

Comment:
why is the value changed?

Reply:
To simplify the test I added a check in the test extension's `ComponentStatusChanged` function to only record events for the component the test cares about.  This changed the number or recorded sources from 5 to 2.